modules/monitoring-creating-alert-routing-for-user-defined-projects.adoc

```diff
@@ -11,8 +11,13 @@ If you are a non-administrator user who has been given the `alert-routing-edit`
 
 .Prerequisites
 
+ifndef::openshift-dedicated,openshift-rosa[]
 * A cluster administrator has enabled monitoring for user-defined projects.
 * A cluster administrator has enabled alert routing for user-defined projects.
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+* Alert routing has been enabled for user-defined projects.
+endif::openshift-dedicated,openshift-rosa[]
 * You are logged in as a user that has the `alert-routing-edit` role for the project for which you want to create alert routing.
 * You have installed the OpenShift CLI (`oc`).
```

Comment:
Content in this module looks good and is rendering correctly across the distros..